sec/review.tex

```diff
@@ -246,6 +247,55 @@ \subsubsection{Centroid-based clustering}\label{subsubsec:kmeans}
 agnostic to its distance measure, and removes the requirement for the attribute
 space to be a metric space at all.
 
+\begin{table}
+    \input{tex/datasets.tex}
+    \caption{A summary of the synthetic datasets}\label{tab:datasets}
+\end{table}
+
+Figure~\ref{fig:kmeans_examples} demonstrates the nuances of centroid-based
+clustering discussed in this section, and shows how the \(k\)-means algorithm
+performs on three synthetic datasets. A brief summary of these datasets is
+provided in Table~\ref{tab:datasets}. The datasets were created using the Python
+library Scikit-learn~\cite{scikit-learn}, and the source code to generate them
+(and the plots included in this survey) has been archived online
+under~\doi{doi-info}.  % TODO archive online
```

Comment:
```suggestion
library Scikit-learn~\cite{scikit-learn}, and the source code to generate them
(and the plots included in this survey) is available at the GitHub repository for 
this manscript \url{}  % TODO Add GitHub repo
and has also been archived online
under~\doi{doi-info}.  % TODO archive online
```

Reply:
Cool -- I didn't see this before I commented.

To reiterate [my comment](https://github.com/drvinceknight/literature-review/pull/1#issuecomment-743097530) below: you would leave the source code (a Jupyter notebook) on GitHub, and only archive the plots along with a useful `README`?